bundles/eclipselink/pom.xml

```diff
@@ -185,6 +185,14 @@
         </dependency>
 
         <!-- included artifacts -->
+        <dependency>
+            <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.parent</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <classifier>common-license.zip</classifier>
```

Comment:
no need to have `.zip` suffix in the `classifier`, it is already present in the `type`